modules/hostthermaldiscovery/hostthermaldiscovery.go

```diff
@@ -0,0 +1,273 @@
+/* HostThermalDiscovery.go: This module performs CPU Thermal discovery of HPC nodes through in-band (through OS) mechanism.
+ *
+ * Author: Ghazanfar Ali, ghazanfar.ali@ttu.edu; Kevin Pelzel <kevinpelzel22@gmail.com>; J. Lowell Wofford <lowell@lanl.gov>
+ *
+ * This software is open source software available under the BSD-3 license.
+ * Copyright (c) 2018, Triad National Security, LLC
+ * See LICENSE file for details.
+ */
+
+//go:generate protoc -I ../../core/proto/include -I proto --go_out=plugins=grpc:proto proto/pxe.proto
```

Comment:
remove this comment or replace with the correct file (not proto/pxe.proto)

Reply:
Fixed by commit 350d25b37ab43747487a4510feba5b17e9fce196